src/swmmanywhere/metric_utilities.py

```diff
@@ -12,13 +12,13 @@
 import cytoolz.curried as tlz
 import geopandas as gpd
 import joblib
```

Comment:
this should be down with the `swmmanywhere` imports no? `ruff` is moving this in `pre-commit` on my machine back down to `swmmanywhere` but `precommit.ci` is moving it back on here... @cheginit do you have any ideas here as @dalonsoa is unavailable at the mo

Reply:
this should be for the `import netcomp` line

Reply:
Is this about `isort`? If so, you need to specify what are the first-party packages.

```toml
lint.isort.known-first-party = [
  "swmmanywhere",
  "netcomp",
]
```

Reply:
It shouldn't include `src`, just the name of the package. The pre-commit error sounds like an issue with the service.

Reply:
OK - shall I just abandon `precommit.ci` and return it to what we had previously for now?

Reply:
oh lol it has passed now - maybe was a one off

Reply:
Yeah, it sometimes happens, so don't sweat it 😄